adal/authentication_context.py

```diff
@@ -67,8 +70,20 @@ def __init__(self, authority, validate_authority=None, cache=None):
         self._oauth2client = None
         self.correlation_id = None
         env_value = os.environ.get('ADAL_PYTHON_SSL_NO_VERIFY')
+        if api_version is not None:
+            warnings.warn(
+                """The default behavior of including api-version=1.0 on the wire
+                is now deprecated.
+                Future version of ADAL will change the default value to None.

```

Comment:
My understanding is we will just remove this parameter in future, as `adal` is not a protocol library. If yes, let us change warning words accordingly such as 
`Future version of ADAL will remove this parameter and use more recent and better version`